server/src/main/java/org/hyperskill/database/serverside/Server.java

```diff
@@ -1,7 +1,49 @@
 package org.hyperskill.database.serverside;
 
+import java.io.*;
+import java.net.*;
+import java.util.*;
+
+//java -cp target/...jar com.kkk.app
+
 public class Server {
+    private static final int PORT = 34522;
+
     public static void main(String[] args) {
-        System.out.println("Hello World!");
+        try (ServerSocket server = new ServerSocket(PORT)) {
+            while (true) {
+                Session session = new Session(server.accept());
+                session.run(); // it does not block this thread
+            }
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+    }
+}
+
+class Session {
+    private final Socket socket;
+
+    public Session(Socket socketForClient) {
+        this.socket = socketForClient;
+    }
+
+    public void run() {
+        try (
+            DataInputStream input = new DataInputStream(socket.getInputStream());
+            DataOutputStream output = new DataOutputStream(socket.getOutputStream())
+        ) {
+          System.out.println("Server started!");
+            while(true) {
+                String msg = input.readUTF();
+
+                  System.out.println("Recieved: " + msg);
+                  output.writeUTF(msg);
+                  System.out.println("Sent: " + msg);
```

Comment:
Wrong input and output messages format.
You only send integers, but the task is to send "Give me a record # N" from Client and "A record # N was sent!" from Server